deploy-manage/deploy/cloud-enterprise/deploy-small-installation.md

```diff
@@ -1,9 +1,13 @@
 ---
+applies_to:
+  deployment:
+    ece: all
 mapped_pages:
   - https://www.elastic.co/guide/en/cloud-enterprise/current/ece-install-small-cloud.html
+  - https://www.elastic.co/guide/en/cloud-enterprise/current/ece-install-small-onprem.html
 ---
 
-# Deploy a small installation cloud [ece-install-small-cloud]
+# Deploy a small installation [ece-install-small]
 
 The type of installation is recommended for development, test, and small-scale use cases. You need:
```

Comment:
in the before you start section, you should link up to the install overview so people can do the prerequisites.

Reply:
done!